style.css

```diff
@@ -286,3 +287,101 @@ h6 {
   row-gap: 50px;
   grid-template-rows: repeat(6, 1fr);
 }
+
+.formy {
+  margin-top: 2rem;
+  width: 375px;
+  height: 779px;
+  background-image: url("./image/form2.png"), url("./image/contactme.png");
```

Comment:
- [ ] Well done on adding the image, but it is not displaying yet, I think it would be fine if you correct the path to your image and resubmit your work, this is how your current design looks like

<img width="719" alt="what" src="https://user-images.githubusercontent.com/77843050/206230267-7ce0d39d-6c51-4431-96c2-0290dab38ed9.png">